docs/developer-docs/multi-chain/chain-key-tokens/ckerc20/overview.mdx

```diff
@@ -0,0 +1,102 @@
+---
```

Comment:
I would suggest adding some examples of current ckERC20 tokens, such as ckUSDC and ckLINK. We've had several users ask about the ckUSDC docs, so we should reference at least ckUSDC somewhere in this overview so we have something to point them to.

Reply:
Good point. I'll add a short paragraph.